glide.yaml

```diff
@@ -5,8 +5,17 @@ import:
 - package: github.com/spf13/pflag
   version: 583c0c0531f06d5278b7d917446061adc344b5cd
 - package: k8s.io/helm
-  version: v2.9.0
-#taken from k8s.io/helm hlide.yaml
+  version: v2.10.0
+  subpackages:
+  - pkg/downloader
+  - pkg/getter
+  - pkg/helm/environment
+  - pkg/helm/helmpath
+  - pkg/nstrvals
+  - pkg/helm
+  - pkg/tlsutil
+  - pkg/proto/hapi/release
```

Comment:
I'm not entirely sure if this is good thing to do, but tried my best not to install unnecessary subpackages to vendor/ while building the project.